README.md

```diff
@@ -31,17 +31,16 @@ can be used as a starting point.
 ### Authentication
 
 The mongodbatlas library does not directly handle authentication. Instead, when
-creating a new client, pass an http.Client that can handle authentication for
-you. The easiest and recommended way to do this is using the [digest](https://github.com/Sectorbob/mlab-ns2/gae/ns/digest)
+creating a new client, pass an http.Client that can handle Digest Access authentication for
+you. The easiest way to do this is using the [digest](https://github.com/mongodb-forks/digest)
 library, but you can always use any other library that provides an `http.Client`.
-If you have a private and public API token pair, you can
-use it with the digest library using:
+If you have a private and public API token pair, you can use it with the digest library using:
```

Comment:
Suggest adding a link to how to create the pair - we get a lot of user comments about how it's hard to figure this out.  https://docs.atlas.mongodb.com/configure-api-access/ is a pretty solid page as it has the UI instructions and links to the API as well.